gufe/proteincomponent.py

```diff
@@ -1,6 +1,12 @@
 # This code is part of OpenFE and is licensed under the MIT license.
 # For details, see https://github.com/OpenFreeEnergy/gufe
 from rdkit import Chem
+try:
```

Comment:
This is more of a thought, but if there isn't a package for optional importing that works maybe we should consider making one at some point (would be useful for MDA too).